cmd/rukpakctl/cmd/content.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright © 2022 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+)
+
+type options struct {
+	k8sclient *kubernetes.Clientset
+	rtclient  runtimeclient.Client
+	namespace string
+}
+
+var opt options
+
+var contentCmd = &cobra.Command{
+	Use:   "content <bundle name>",
+	Short: "display contents of the specified bundle.",
+	Long:  `display contents of the specified bundle.`,
+	Args: func(cmd *cobra.Command, args []string) error {
+		if len(args) < 1 {
+			return errors.New("requires 2 argument: <bundle name>")
+		}
+		return nil
+	},
+	Run: func(cmd *cobra.Command, args []string) {
+		kubeconfig, err := cmd.Flags().GetString("kubeconfig")
+		if err != nil {
+			fmt.Printf("failed to find kubeconfig location: %+v\n", err)
+			return
+		}
+		// use the current context in kubeconfig
+		config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+		if err != nil {
+			fmt.Printf("failed to find kubeconfig location: %+v\n", err)
+			return
+		}
+
+		scheme := runtime.NewScheme()
+		err = rukpakv1alpha1.AddToScheme(scheme)
+		if err != nil {
+			fmt.Printf("failed to add schema: %+v\n", err)
+			return
```

Comment:
I wonder, could we the scheme in an init somewhere at the root-level of the tool? So we don't have to add to the scheme in each command separately.

Reply:
I'll move it to the rootCmd.